auto_sizing/workflows/run.yaml

```diff
@@ -54,3 +58,13 @@ spec:
         factor: 2
         maxDuration: "5m"
 
+  - name: export-results
+    inputs:
+    container:
+      image: gcr.io/moz-fx-data-experiments/auto_sizing:latest
+      command: [
+        auto_sizing, export-aggregate-results, 
+        "--project_id={{workflow.parameters.project_id}}",
+        "--bucket={{workflow.parameters.bucket}}"
+      ]
+    activeDeadlineSeconds: 600   # terminate container template after 10 minutes
```

Comment:
Just noticed this -- what's the goal of this approach (vs the retryStrategy or other approaches to generally clean up)?

Reply:
Decision made to leave this in as it works fine for [Jetstream's export](https://github.com/mozilla/jetstream/blob/main/jetstream/workflows/run.yaml#L118).